main-settings/src/main/scala/sbt/Def.scala

```diff
@@ -81,15 +81,7 @@ object Def extends BuildSyntax with Init[Scope] with InitializeImplicits:
   def showFullKey(keyNameColor: Option[String]): Show[ScopedKey[_]] =
     Show[ScopedKey[_]]((key: ScopedKey[_]) => displayFull(key, keyNameColor))
 
-  @deprecated("Use showRelativeKey2 which doesn't take the unused multi param", "1.1.1")
   def showRelativeKey(
-      current: ProjectRef,
-      multi: Boolean,
-      keyNameColor: Option[String] = None
-  ): Show[ScopedKey[_]] =
-    showRelativeKey2(current, keyNameColor)
-
-  def showRelativeKey2(
```

Comment:
Should we keep the name `showRelativeKey2` for cross-compatibility with sbt 1.x

Reply:
Indeed there is a trade off between facilitating cross-compatibility with sbt 1.x and doing all the breaking changes now...

I guess, we could keep `showRelativeKey2` for backwards compatibilty, but mark it as deprecated...

Reply:
Indeed there is a trade off between facilitating cross-compatibility with sbt 1.x and doing all the breaking changes now...

I guess, we could keep `showRelativeKey2` for backwards compatibilty, but mark it as deprecated...

Reply:
I added those back with an `inline` keyword.